bakery/src/tasks/link-rex.js

```diff
@@ -0,0 +1,75 @@
+const dedent = require('dedent')
+
+const { constructImageSource } = require('../task-util/task-util')
+
+const task = (taskArgs) => {
+  const { inputSource, inputPath, contentSource: maybeContentSource } = taskArgs
+  const imageDefault = {
+    name: 'openstax/cops-bakery-scripts',
+    tag: 'trunk'
+  }
+  const imageOverrides = taskArgs != null && taskArgs.image != null ? taskArgs.image : {}
+  const imageSource = constructImageSource({ ...imageDefault, ...imageOverrides })
+  const contentSource = maybeContentSource != null ? maybeContentSource : 'archive'
+  const bookSlugsUrl = 'https://raw.githubusercontent.com/openstax/content-manager-approved-books/master/approved-book-list.json'
```

Comment:
Can we take advantage of the fact that for git books `link-single` will populate `data-book-slug` for all other-books links per [these docs](https://openstax.atlassian.net/wiki/spaces/CE/pages/1759707137/Pipeline+Pipeline+Task+Definitions#Git-Links) (e.g. I think for `git` all the information you need to create the REX link is already in the XHTML) to avoid downloading the ABL for git books in this task?

Reply:
yes, thank you!